samples/matter/weather_station/README.rst

```diff
@@ -0,0 +1,117 @@
+.. _matter_weather_station_sample:
+
+Matter: Weather station
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This weather station sample demonstrates the usage of the :ref:`Matter <ug_matter>` (formerly Project Connected Home over IP, Project CHIP) application layer to build a weather station device that lets you remotely gather different kind of data, such as temperature, air pressure, and relative humidity, using the device sensors.
+This device works as a Matter accessory device, meaning it can be paired and controlled remotely over a Matter network built on top of a low-power, 802.15.4 Thread network.
+You can use this sample as a reference for creating your own application.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: thingy53_nrf5340
+
+To program and gather logs from the Thingy:53 device, you need the external J-Link programmer.
+If you own an nRF5340 DK that has an on-board J-Link programmer, you can also use it for this purpose.
+
+For remote testing scenarios, if you want to commission the weather station device and :ref:`control it remotely <matter_weather_station_sample_remote_control>` through a Thread network, you also need the following:
+
+* A Matter controller device :ref:`configured on PC or smartphone <ug_matter_configuring>`.
+
+.. note::
+    |matter_gn_required_note|
+
+Overview
+********
+
+The sample can be tested in the following ways:
```

Comment:
How about:

> The sample uses single button for controlling the device state.
It can be tested remotely over the Thread protocol, which requires additional Matter controller device that can be configured either on PC or mobile (for remote testing in a network).
The weather station device is periodically performing temperature, air pressure and relative humidity measurements, which results can be read using the Matter controller.
It communicates with the weather station device over Matter protocol utilizing Zigbee Cluster Library in order to describe data measurements within the proper clusters. 
To enable testing you need to complete :ref:`building and running the sample <matter_lock_sample_remote_control>`.
?